addons/sale_management/models/sale_order.py

```diff
@@ -62,6 +65,12 @@ def _compute_validity_date(self):
             if validity_days > 0:
                 order.validity_date = fields.Date.context_today(order) + timedelta(validity_days)
 
+    @api.depends('sale_order_template_id')
+    def _compute_journal_id(self):
+        super()._compute_require_signature()
```

Comment:
This is strange. If you depends on `require_signature`, you should add it to the dependences. :thinking: